### PR TITLE
#29-tag table

### PR DESCRIPTION
**Pull request by shakilsmash**

### CODE REVIEW

File: src/main/java/com/ronniegnr/kothamala/domain/Tag.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "tag")
+public class Tag extends AbstractAuditingEntity implements Serializable {
+
+    private static final long serialVersionUID = 1;
+
+    public enum Status { ACTIVE, INACTIVE, BANNED }
```
Comment (ronniegnr) [inline src/main/java/com/ronniegnr/kothamala/domain/Tag.java]: No need for BANNED status.

File: src/main/java/com/ronniegnr/kothamala/domain/Tag.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    private long id;
+    private String name;
+    private Status status;
+
+    private Set<Authority> authorities = new HashSet<>();
```
Comment (ronniegnr) [inline src/main/java/com/ronniegnr/kothamala/domain/Tag.java]: Remove authorities property.

File: src/main/java/com/ronniegnr/kothamala/domain/Tag.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.hibernate.validator.constraints.Email;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.io.Serializable;
+import java.util.HashSet;
+import java.util.Set;
+
```
Comment (ronniegnr) [inline src/main/java/com/ronniegnr/kothamala/domain/Tag.java]: Remove unused imports.

Comment (shakilsmash) [inline src/main/java/com/ronniegnr/kothamala/domain/Tag.java]: fixed

File: src/main/java/com/ronniegnr/kothamala/domain/Tag.java (hunk 1 above)
Comment (shakilsmash) [inline src/main/java/com/ronniegnr/kothamala/domain/Tag.java]: fixed

File: src/main/java/com/ronniegnr/kothamala/domain/Tag.java (hunk 2 above)
Comment (shakilsmash) [inline src/main/java/com/ronniegnr/kothamala/domain/Tag.java]: fixed

**Pull request merged**